resources/lang/en/events/lead_ceremony_sc.json

```diff
@@ -677,7 +648,7 @@
       "rank": ["warrior", "elder", "medicine cat", "mediator", "apprentice", "deputy"],
       "life_giving": [
         {
-          "text": "r_c had to get m_c's attention once or twice during the ceremony, but eventually gives the new c_n leader a life for [virtue].",
+          "text": "r_c has to get m_c's attention once or twice during the ceremony, but eventually gives the new c_n leader a life for [virtue].",
```

Comment:
Mmh. I know what this is trying to convey, but I just really don't think it comes across well.